pkg/providers/instancetype/instancetypes.go

```diff
@@ -136,7 +136,7 @@ func (p *Provider) createOfferings(sku *skewer.SKU, zones sets.Set[string]) []cl
 		onDemandPrice, onDemandOk := p.pricingProvider.OnDemandPrice(*sku.Name)
 		spotPrice, spotOk := p.pricingProvider.SpotPrice(*sku.Name)
 		availableOnDemand := onDemandOk && !p.unavailableOfferings.IsUnavailable(*sku.Name, zone, corev1beta1.CapacityTypeOnDemand)
-		availableSpot := spotOk && !p.unavailableOfferings.IsUnavailable(*sku.Name, zone, corev1beta1.CapacityTypeSpot)
+		availableSpot := spotOk && !p.unavailableOfferings.IsUnavailable(*sku.Name, zone, corev1beta1.CapacityTypeSpot) && !p.unavailableOfferings.Has(sdkerrors.LowPriorityQuotaExceededTerm)
```

Comment:
Can the new check be added within IsUnavailable? Simpler and easier to use, better encapsulation / don't need Has ...

Reply:
yep

Reply:
Sorry I did not make myself clear; I meant adding th check within IsUnavailable _implementation_ - so this call would not even need to change.